docs/installation.md

```diff
@@ -63,3 +73,9 @@ brew install uv
 uv provides a Docker image at [`ghcr.io/astral-sh/uv`](https://github.com/astral-sh/uv/pkgs/container/uv).
 
 See our guide on [using uv in Docker](./guides/docker.md) for more details.
+
+## GitHub Releases
+
+uv artifacts can be downloaded directly from [GitHub Releases](https://github.com/astral-sh/uv/releases).
+
+Each release page includes binaries for all support platforms as well as instructions for using the standalone installer via `github.com` instead of `astral.sh`.
```

Comment:
```suggestion
Each release page includes binaries for all supported platforms as well as instructions for using the standalone installer via `github.com` instead of `astral.sh`.
```